lib/compat/wordpress-6.6/rest-api.php

```diff
@@ -29,3 +29,36 @@ function wp_api_template_access_controller( $args, $post_type ) {
 	}
 }
 add_filter( 'register_post_type_args', 'wp_api_template_access_controller', 10, 2 );
+
+
+/**
+ * Registers additional fields for WP_REST_Themes_Controller class.
+ * https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php
+ */
+function gutenberg_register_wp_theme_directory_uri_field() {
```

Comment:
This is the test. 

Is there another way to get the theme directory to the editor?